### PR TITLE
Replace k/security with k/committee-security-response

### PR DESCRIPTION
**Pull request by PushkarJ**
/sig security
/committee security-response

/hold for https://github.com/kubernetes/org/issues/2967

### CODE REVIEW

File: content/en/releases/release-managers.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ The responsibilities of each role are described below.
 
 ### Security Embargo Policy
 
-Some information about releases is subject to embargo and we have defined policy about how those embargos are set. Please refer [Security Embargo Policy](https://github.com/kubernetes/security/blob/master/private-distributors-list.md#embargo-policy) here for more information.
+Some information about releases is subject to embargo and we have defined policy about how those embargos are set. Please refer [Security Embargo Policy](https://github.com/kubernetes/committee-security-response/blob/master/private-distributors-list.md#embargo-policy) here for more information.
```
Comment (reylejano) [inline content/en/releases/release-managers.md:34]: A few nits:
- plural for embargo is embargoes according to [merriam-webster](https://www.merriam-webster.com/dictionary/embargo)
- word usage
```suggestion
Some information about releases is subject to embargo and we have defined policy about how those embargoes are set. Please refer to the [Security Embargo Policy](https://github.com/kubernetes/committee-security-response/blob/master/private-distributors-list.md#embargo-policy) for more information.
```

**Comment by PushkarJ** [comment]
/hold cancel

[Repo](https://github.com/kubernetes/org/issues/2967#issuecomment-921946327) has been renamed

File: content/en/releases/release-managers.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ The responsibilities of each role are described below.
 
 ### Security Embargo Policy
 
-Some information about releases is subject to embargo and we have defined policy about how those embargos are set. Please refer [Security Embargo Policy](https://github.com/kubernetes/security/blob/master/private-distributors-list.md#embargo-policy) here for more information.
+Some information about releases is subject to embargo and we have defined policy about how those embargoes are set. Please refer to the [Security Embargo Policy](https://github.com/kubernetes/committee-security-response/blob/master/private-distributors-list.md#embargo-policy) for more information.
```
Comment (mrbobbytables) [inline content/en/releases/release-managers.md:34]:
```suggestion
Some information about releases is subject to embargo and we have defined policy about how those embargoes are set. Please refer to the [Security Embargo Policy](https://github.com/kubernetes/committee-security-response/blob/main/private-distributors-list.md#embargo-policy) for more information.
```

default branch has now been renamed 👍

**Comment by reylejano** [comment]
Preview of updated page and section: 
https://deploy-preview-29693--kubernetes-io-main-staging.netlify.app/releases/release-managers/#security-embargo-policy

**Comment by reylejano** [comment]
- kubernetes/org#2967 is closed
- repo has been renamed
- preview looks good and the updated link works

/lgtm
/approve

**Pull request merged**